llvm/tools/llvm-mc/llvm-mc.cpp

```diff
@@ -240,6 +242,24 @@ static cl::opt<ActionType> Action(
                           "Colored disassembly of strings of hex bytes")),
     cl::cat(MCCategory));
 
+static cl::opt<unsigned>
+    BenchmarkNumRuns("benchmark-num-runs",
```

Comment:
nit: `num-benchmark-runs`? Number of benchmark runs

Reply:
Done